CMakeLists.txt

```diff
@@ -60,6 +60,9 @@ option(LLAMA_FMA                    "llama: enable FMA"
 # 3rd party libs
 option(LLAMA_ACCELERATE             "llama: enable Accelerate framework"                    ON)
 option(LLAMA_OPENBLAS               "llama: use OpenBLAS"                                   OFF)
+if (LLAMA_OPENBLAS)
```

Comment:
Why not use FindPackage for OPENBLAS?

Reply:
find_package is used below, on line 115. find_package(BLAS) does not return the include path, only the path to the library. The current CMakeLists doesn't add -I/usr/include/openblas and the generated makefile are unable to build llama on Fedora.
`
/home/vincenzo/Projects/llama.cpp/ggml.c:145:10: fatal error: cblas.h: File o directory non esistente
  145 | #include <cblas.h>
`